src/processing/handlers/ObjectHandler.java

```diff
@@ -15,8 +16,9 @@
 
 public class ObjectHandler implements INotificationHandler{
 	
-	public void onMessageReceived(String message) {
+	public static void handleBoardObject(String message) {
 		
+
 		try {
 				BoardObject boardObject = (BoardObject)Serialize.deSerialize(message);
 				boardObject.getOperation();
```

Comment:
What are you doing here...?

Reply:
You must also call the actual operation after retrieving what operation is to be performed

Reply:
You must also perform the operation that you retrieved

Reply:
Also don't forget to check if the object is sent from the same user (by looking at user ID), else you would perform it twice....

Reply:
This is satchit's file, he will complete it, it's not completed yet.